server/pkg/server/watchstate/watchstate.go

```diff
@@ -101,16 +99,16 @@ func (w *WatchState) send(item *localnetv1.OpItem) {
 }
 
 func (w *WatchState) sendSet(set localnetv1.Set, path string, m proto.Message) {
-	w.pb.Reset()
```

Comment:
wait , no more reset ? whys that

Reply:
adding a reset at this point will clear new requests..

Reply:
this reset in particular is from the buffer and not the message..
adding a `proto.Reset(m)` at this point will clear new request so they come up as empty.